android/beagle/src/main/java/br/com/zup/beagle/android/context/operations/operation/Operation.kt

```diff
@@ -0,0 +1,18 @@
+package br.com.zup.beagle.android.context.operations.operation
+
+import br.com.zup.beagle.android.context.operations.strategy.BaseOperation
+import br.com.zup.beagle.android.context.operations.strategy.Operations
+
+/**
+ * @param operationToken Store the operation token that will be solved to update the principal operation with result
+ *
+ * @param operationStrategy The OperationType is the strategy of all operations
+ * @see BaseOperation
+ *
+ * @param operationValue The value that will be solved by strategy above
+ *
+ * **/
+
+data class Operation (val operationToken: String,
```

Comment:
Break each parameter in a new line:
```suggestion
data class Operation (
    val operationToken: String,
```

Reply:
Done